examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/ImageViewer.common.kt

```diff
@@ -84,13 +95,18 @@ internal fun ImageViewerWithProvidedDependencies(
                     pictures = pictures,
                     memoryPage = page,
                     onSelectRelatedMemory = { picture ->
-                        navigationStack.push(MemoryPage(mutableStateOf(picture)))
+                        navigationStack.push(MemoryPage(pictures.indexOf(picture)))
```

Comment:
Can we use picture index here instead of linearly searching index?

Reply:
Oooops, wrong name. I want to make it as previewPictureIndex: Int
But for now it have type previewPictureIndex: Picture

I will change it to index

Reply:
Done